manifests/07_deployment.yaml

```diff
@@ -36,6 +36,9 @@ spec:
           name: config
         - mountPath: /var/run/secrets/serving-cert
           name: serving-cert
+        - mountPath: /etc/pki/ca-trust/extracted/pem
```

Comment:
This will not get included in the root CAs that our go code is using.  So what is the point?

Reply:
From what I gather this is one of the root cert stores that golang searches to get its system trust store.

We have pieces of code in oauth-server that seem to be using default HTTP transport if the user does not provide their own CA.

If they however do specify a CA in a proxied environment, without NO_PROXY adjustments, they are either not aware that proxying is in place, which might mean that someone else is trying to steal all their traffic, or they don't know what they are doing. I'll gladly have the code fail in both cases :man_shrugging:

Reply:
Pfft, the logic for getting CA data we're using seems a bit arbitrary (not saying wrong, though), I'll need to have a better look.

Reply:
If I got it right, this would only be used in the case where `ca` field is not specified for an OpenID IdP, which is exactly the single one use where we want it. If `ca` is configured and `NO_PROXY` is not set, it should not be our problem.

Reply:
Your attempt to reload system roots will not work without a restart of the entire pod.